src/dyninst/test1_1_mutatee.c

```diff
@@ -86,7 +86,7 @@ void test1_1_func1_1() {
 int test1_1_mutatee() {
   dprintf("Address of func1_1 is %p, calling func1_1\n", &test1_1_func1_1);
   test1_1_func1_1();
-  dprintf("Returned to mutatee main function from instrumented functions.\n", &test1_1_func1_1);
+  dprintf("Returned to mutatee main function from instrumented functions.\n");
```

Comment:
I felt like the intention is to report the address of test1_1_func_1 as well, so maybe we should add %p instead of deleting the extra parameter?

Reply:
Since line 87 prints the address, I think this one was a copy/paste error.